src/textual/reactive.py

```diff
@@ -169,53 +159,41 @@ def _reset_object(cls, obj: object) -> None:
         getattr(obj, "__watchers", {}).clear()
         getattr(obj, "__computes", []).clear()
 
-    def __set_name__(self, owner: Type[MessageTarget], name: str) -> None:
-
+    def __set_name__(self, owner: Reactable, name: str) -> None:
         # Check for compute method
         if hasattr(owner, f"compute_{name}"):
             # Compute methods are stored in a list called `__computes`
-            try:
-                computes = getattr(owner, "__computes")
-            except AttributeError:
+            if not hasattr(owner, "__computes"):
                 computes = []
                 setattr(owner, "__computes", computes)
+            else:
+                computes = getattr(owner, "__computes")
             computes.append(name)
 
         # The name of the attribute
         self.name = name
         # The internal name where the attribute's value is stored
         self.internal_name = f"_reactive_{name}"
-        default = self._default
-        setattr(owner, f"_default_{name}", default)
 
     def __get__(self, obj: Reactable, obj_type: type[object]) -> ReactiveType:
         _rich_traceback_omit = True
 
         self._initialize_reactive(obj, self.name)
-
-        value: ReactiveType
-        compute_method = getattr(self, f"compute_{self.name}", None)
-        if compute_method is not None:
-            old_value = getattr(obj, self.internal_name)
-            value = getattr(obj, f"compute_{self.name}")()
-            setattr(obj, self.internal_name, value)
-            self._check_watchers(obj, self.name, old_value)
-        else:
-            value = getattr(obj, self.internal_name)
+        value: ReactiveType = getattr(obj, self.internal_name)
         return value
 
     def __set__(self, obj: Reactable, value: ReactiveType) -> None:
         _rich_traceback_omit = True
 
         self._initialize_reactive(obj, self.name)
+        current_value = getattr(obj, self.internal_name)
         name = self.name
-        current_value = getattr(obj, name)
         # Check for validate function
         validate_function = getattr(obj, f"validate_{name}", None)
         # Call validate
         if callable(validate_function):
             value = validate_function(value)
-        # If the value has changed, or this is the first time setting the value
+        # If the value has changed, or "always_update" is set
```

Comment:
old comment (?); part of what led me to my guess about `_NOT_SET` which I deleted above.

Reply:
I think that comment is redundant anyway. Pretty much says the following line in English. Best to remove I think.